base/libgit2/tree.jl

```diff
@@ -18,7 +18,9 @@ end
 "Returns a file name, as a `String`, of a tree entry."
 function filename(te::GitTreeEntry)
     str = ccall((:git_tree_entry_name, :libgit2), Cstring, (Ptr{Void},), te.ptr)
-    str == C_NULL && return ""
+    str == C_NULL && throw(Error.GitError(Error.Tree,
+                                          LibGit2.Error.ENOTFOUND,
+                                          "Entry file name is not found"))
```

Comment:
I meant "<subject> not found", just drop the "is" - it's still useful to differentiate what wasn't found